packages-hybrid/hybrid-mobile-actions/scripts/exportModule.ts

```diff
@@ -0,0 +1,159 @@
+/* eslint no-console: 0 */
+import { promises as fs, existsSync, createWriteStream, mkdirSync } from "fs";
+import { join } from "path";
+import { get } from "http";
+import { decompress } from "targz";
+import { spawnSync } from "child_process";
+
+const cwd = process.cwd();
+const buildPath = join(cwd, "dist/build/");
+
+let mendixVersion = "";
+let moduleVersion = "";
+let projectPath = "";
+let moduleName = "";
+let repoMxBuildUrl = "";
+const monoPath = "/Library/Frameworks/Mono.framework/Versions/Current/Commands";
+// For local tests of unreleased mx versions: set useLocaleUtils true, and path to Mendix folder
+const useLocaleUtils = false;
+const localUtilPath = "/Volumes/[C] Windows10Enterprise/Program Files/Mendix/";
```

Comment:
Maybe we should add a local config file for `localUtilPath` and `monoPath`, otherwise we will need to change with a big chance of commit changes in this file